### PR TITLE
[docs] Minor doc edits (phrasing, formatting, etc.)

### PR DESCRIPTION
**Pull request by Zearin**
## Summary

Minor edits (mostly formatting and phrasing tweaks) to the files in `docs/`.

### CODE REVIEW

File: docs/cucumber.md [hunk 1]
```diff
@@ -1,36 +1,64 @@
 #Cucumber:
  
- The first question that would rise for a beginner would be 
+The first questions for a beginner are usually something like:
  
   *1. What is Cucumber?*
   *2. How Cucumber works with BDD?*
```
Comment (olleolleolle) [inline docs/cucumber.md]: I have one change request; the question posed here isn't the same as the (better) heading you edited below.

Proposal: Change the question 2 to read "How does Cucumber work with BDD?"

Comment (techosi) [inline docs/cucumber.md]: Agreed and sound very good.

**Comment by olleolleolle** [comment]
An issue with all this is: documentation edits in this repository are going away.

The push to build documentation will be continued in the repo [cucumber/docs.cucumber.io](https://github.com/cucumber/docs.cucumber.io).

There's a branch called `sw-reorg` which has 70+ commits on top of its master, and @plaindocs is leading the charge to get that finished.

@Zearin Can you create this change as a PR on top of `sw-reorg`? That way, we will not lose these changes going forward. It's a huge effort, and I understand if that feels like too much. But now you know why I didn't move to merge this as soon as possible.

File: docs/cucumber.md (hunk 1 above)
Comment (Zearin) [inline docs/cucumber.md]: 👍 Done

**Comment by Zearin** [comment]
> @Zearin Can you create this change as a PR on top of sw-reorg? That way, we will not lose these changes going forward. It's a huge effort, and I understand if that feels like too much. But now you know why I didn't move to merge this as soon as possible.

(Sorry for the slow response!)

I’ll see if I can figure out how to do the rebase.  (My git skills outside of the common operations are limited.)

Stand by…

**Comment by Zearin** [comment]
Looks like most of the files I edited on the `sw-reorg` branch were deleted.  A few of my changes survived.

I’m going to go through the new documentation files in `sw-reorg` and do a new set of edits.  

Any issues with this?

**Comment by plaindocs** [comment]
Hi @Zearin that would be awesome! Apologies for not catching your PR before it got to this stage.

**Comment by olleolleolle** [comment]
@Zearin Super-thanks for doing the deep dive!

**Comment by Zearin** [comment]
The saga continues at cucumber/docs.cucumber.io/pull/20 …

**Pull request closed without merging**